dialogflow/spec/spec_helper.rb

```diff
@@ -0,0 +1,50 @@
+# Copyright 2018 Google, Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+def get_entity_type_ids project_id:, display_name:
+  # project_id = "Your Google Cloud project ID"
+  # display_name = "Existing Display Name"
+  
+  require "google/cloud/dialogflow"
+
+  entity_types_client = Google::Cloud::Dialogflow::EntityTypes.new
+  parent = entity_types_client.class.project_agent_path project_id
+
+  entity_types = entity_types_client.list_entity_types parent
+
+  selected_entity_types = entity_types.select { |entity_type| entity_type.display_name == display_name }
+
+  entity_type_ids = selected_entity_types.map { |entity_type| entity_type.name.split("/").last }
+
+  return entity_type_ids
```

Comment:
You should never use `return` in Ruby

(As a general rule. Unless short-circuiting.)

Reply:
done